examples/src/main/r/ml/glm.R

```diff
@@ -44,8 +44,9 @@ gaussianGLM2 <- glm(label ~ features, gaussianDF, family = "gaussian")
 summary(gaussianGLM2)
 
 # Fit a generalized linear model of family "binomial" with spark.glm
-training2 <- read.df("data/mllib/sample_binary_classification_data.txt", source = "libsvm")
-df_list2 <- randomSplit(training2, c(7,3), 2)
+training2 <- read.df("/data/mllib/sample_multiclass_classification_data.txt", source = "libsvm")
```

Comment:
actually, you might need to leave it as relative path, ie. not starting with `/`